internal/policy/helpers_test.go

```diff
@@ -203,12 +198,13 @@ func createTestStateWithDelegatedPolicies(t *testing.T) *State {
 
 	// Create the second level of delegations
 	delegation1Metadata := InitializeTargetsMetadata()
-	delegation1Metadata, err = AddOrUpdateDelegation(delegation1Metadata, "3", []*tuf.Key{gpgKey}, []string{"path1/subpath1/*"})
+
+	delegation1Metadata, err = AddOrUpdateDelegation(delegation1Metadata, "3", []*tuf.Key{key}, []string{"path1/subpath1/*"})
```

Comment:
these paths wouldn't do much anyway, because they don't have the `file:` / `git:` path schemes.

Reply:
perhaps that should be resolved separately though.

Reply:
So for now should it be left as it is, should I remove the file paths alltogheter, or should I format it correctly in the file: or git: format?

Reply:
Can you submit a separate small PR focused on this helper that:
* fixes the paths
* uses gpg keys only in the leaf delegation (i.e. which does not point to another metadata)

It might be easier to merge that first and rebase this one.

Reply:
How would the leaf delegations be allowed to use GPG keys, wouldn't state verification fail?

Reply:
Will be fixed with PR #251

Reply:
Let's reverse changes to the delegated policy here :)

Reply:
done